runtime/compiler/optimizer/StringPeepholes.cpp

```diff
@@ -185,7 +185,7 @@ TR::SymbolReference* TR_StringPeepholes::MethodEnumToArgsForMethodSymRefFromName
                                   "<init>",
                           "<init>"};
 
-   static char* signatures [] =  {        "(Ljava/math/BigDecimal;Ljava/math/BigDecimal;Ljava/math/BigDecimal;Ljava/math/BigDecimal;Ljava/math/BigDecimal;IIII)Ljava/math/BigDecimal;",
+   static const char* signatures [] =  {        "(Ljava/math/BigDecimal;Ljava/math/BigDecimal;Ljava/math/BigDecimal;Ljava/math/BigDecimal;Ljava/math/BigDecimal;IIII)Ljava/math/BigDecimal;",
```

Comment:
I'm guessing the values in each of `classNames`, `methodNames` and `signatures` used to line up.  As long as you're here, may I ask you to have them line up again?